src/main/java/seedu/address/commons/core/Config.java

```diff
@@ -74,13 +65,12 @@ public boolean equals(Object other) {
         return Objects.equals(appTitle, o.appTitle)
                 && Objects.equals(logLevel, o.logLevel)
                 && Objects.equals(userPrefsFilePath, o.userPrefsFilePath)
-                && Objects.equals(taskBookFilePath, o.taskBookFilePath)
-                && Objects.equals(taskBookName, o.taskBookName);
+                && Objects.equals(taskBookFilePath, o.taskBookFilePath);
     }
 
     @Override
     public int hashCode() {
-        return Objects.hash(appTitle, logLevel, userPrefsFilePath, taskBookFilePath, taskBookName);
+        return Objects.hash(appTitle, logLevel, userPrefsFilePath, taskBookFilePath);
```

Comment:
I don't know whether it is a good idea or not to just have `taskBookFilePath` to identify the task book.
So that in our application, we do not suggest the users to have multiple task book right?